### DISCUSSION TITLE
No Thumbnails

### DISCUSSION THREAD

**Discussion by ghost** [Difegue/LANraragi#612] [Q&A] [answered]
Fresh Docker install - No Thumbnails. 

This is the docker-compose file I used to build it, where /mnt/Primus/Media is the top directory to my collection, the first folder divisons are by publisher. The database i put in a config directory. Could it be a permissions issue?

version: "3.3"
services:
  lanraragi:
    image: difegue/lanraragi  
    container_name: lanraragi
    ports:
      - 3000:3000
    volumes:
      - /mnt/Primus/Media:/home/koyomi/lanraragi/content
      - /volume1/docker/appdata/lanraragi/config:/home/koyomi/lanraragi/database
    restart: unless-stopped

**Comment by xiazeyu** [comment]
Is there error or warning from the container? Are books readable from the website?
Use `docker ps` find the container's name, then try `docker logs` please.

**Comment by Difegue** [comment, marked as answer]
In a much simpler fashion, you can directly check the status of thumbnailing jobs in the Minion console under settings:  

![image](https://user-images.githubusercontent.com/8237712/164303041-e18bc4d6-d0ec-4e73-b613-cd04d8474ddb.png)
  
Keep in mind that thumbnails take a little while to show up on first installs, as making them all consumes a fair bit of CPU.  

Also be aware that your content directory must be **writable**, since thumbnails are stored in it by default: You can add an extra mounting point in your docker settings + modify the path in LRR settings if you want to use a secondary folder.  

![image](https://user-images.githubusercontent.com/8237712/164303492-c1923545-483f-462d-a1f7-d7dac1bd4062.png)

**Comment by ghost** [comment]
Solved. Had to make the directory writable. Thanks!
